easybuild/tools/module_generator.py

```diff
@@ -187,7 +187,6 @@ def get_description(self, conflict=True):
             whatis = [description]
 
         lines = [
-            self.MODULE_HEADER.replace('%', '%%'),
```

Comment:
it's moved to `make_module_step`, we add there shebang there to start with, then add the `--modules-header` (if any), then the description